README.rst

```diff
@@ -337,6 +337,27 @@ The following options are supported by both ``delay`` and the task decorator:
   For example, to retry a task 3 times (for a total of 4 executions), and wait
   60 seconds between executions, pass ``retry_method=fixed(60, 3)``.
 
+- ``runner_class``
+
+  If given, a Python class can be specified to influence task running behavior.
+  The runner class should inherit `tasktiger.runner.BaseRunner` and implement
+  the task execution behavior. The default implementation is available in
+  `tasktiger.runner.DefaultRunner`. The following behavior can be achieved:
+
+  - Execute specific code before or after the task is executed (while forked),
```

Comment:
NIT: Should we use the wording "in the forked child process" somewhere in this section to be very clear these runners only operate in the child? I had to think for a moment what _(while forked)_ here meant.

Reply:
Like this? https://github.com/closeio/tasktiger/pull/175/commits/6df51f69455308abcebe19c77a350936ede46ab5

Reply:
Yep